collect_app/src/main/java/org/odk/collect/android/external/InstanceProvider.java

```diff
@@ -197,67 +193,7 @@ public int delete(@NonNull Uri uri, String where, String[] whereArgs) {
 
     @Override
     public int update(@NonNull Uri uri, ContentValues values, String where, String[] whereArgs) {
-        DaggerUtils.getComponent(getContext()).inject(this);
-
-        String projectId = getProjectId(uri);
-        logServerEvent(projectId, AnalyticsEvents.INSTANCE_PROVIDER_UPDATE);
```

Comment:
This analytics event should be deleted as well.

Reply:
Done.